src/clt_least_squares/InputForm.jsx

```diff
@@ -17,191 +17,192 @@ export const InputForm = ({seed, handleSeed, handleGeneratePop, beta,
-                        <NumericField
-                            id={'alpha'}
-                            className={
-                                'form-control form-control-sm numeric-field'}
-                            min={-1}
-                            max={1}
-                            step={0.01}
-                            value={math.round(alpha, 2)}
-                            onChange={handleAlpha} />
-                    </div>
-                    <div className={'form-row'}>
-                        <div style={{ height: '50px', width: '100%'}}>
-                            <Rheostat
-                                min={-100}
-                                max={100}
-                                values={[math.round(alpha * 100, 2)]}
-                                onValuesUpdated={(sliderState) => {
-                                    handleAlpha(
-                                        (sliderState.values[0] * 0.01));
-                                }} />
-                        </div>
-                    </div>
-                    <div className={'form-row'}>
-                        <label htmlFor={'slope'}>
-                            <InlineMath>
-                                {String.raw`\beta_1`}
-                            </InlineMath>
-                        </label>
-                        <span className="help-tooltip"
-                            tabIndex="0"
-                            data-tip
-                            data-for="beta-tt">
-                            <sup>
-                                <i className="fas fa-question-circle"></i>
-                            </sup>
-                        </span>
-                        <ReactTooltip id="beta-tt" event="focus"
-                            eventOff="blur">
-                            <span>Slope of the regression.</span>
-                        </ReactTooltip>
-                        <NumericField
-                            id={'beta'}
-                            className={
-                                'form-control form-control-sm numeric-field'}
-                            min={-1}
-                            max={1}
-                            value={math.round(beta, 2)}
-                            step={0.01}
-                            onChange={handleBeta} />
-                    </div>
-                    <div className={'form-row'}>
-                        <div style={{ height: '50px', width: '100%'}}>
-                            <Rheostat
-                                min={-100}
-                                max={100}
-                                values={[math.round(beta * 100, 2)]}
-                                onValuesUpdated={(sliderState) => {
-                                    handleBeta(
-                                        (sliderState.values[0] * 0.01));
-                                }} />
+        <div className={'sticky-top'}>
```

Comment:
The sticky scroll is only working for me at certain points. I can review this with you tomorrow.

Reply:
Ok cool.  This sticky scroll has caused me some agita - it doesn't work at all on Safari.  Lets look together.